src/actions/actions.test.js

```diff
@@ -207,4 +217,36 @@ describe('actions', () => {
     };
     expect(changeFlag('testFlag', true)).toEqual(expectedAction);
   });
+
+  it('should create an action to toggle display chonky graph', () => {
+    const expectedAction = {
+      type: TOGGLE_DISPLAY_CHONKY_GRAPH,
+      displayChonkyGraph: true
+    };
+    expect(toggleDisplayChonkyGraph(true)).toEqual(expectedAction);
+  });
+
+  it('should create an action to toggle isChonky', () => {
+    const expectedAction = {
+      type: TOGGLE_IS_CHONKY,
+      isChonky: true
+    };
+    expect(toggleIsChonky(true)).toEqual(expectedAction);
+  });
+
+  it('should create an action to update the no. of nodes for the modal display', () => {
```

Comment:
I think maybe clearer to avoid abbreviations and go with the word `count` or `number`

Reply:
thanks @bru5 , I'll update the wording

Reply:
updated in latest commit